fiftyone/utils/complex_yolo/zoo.py

```diff
@@ -44,250 +75,200 @@ class TorchComplexYOLOv3ModelConfig(
     arguments.
 
     Args:
+        config_url: a URL from which the model's config can be downloaded,
+            if necessary
         config_base_filename: the filename in ``fo.config.model_zoo_dir`` in
             which to store the model's config file
-        config_base_url: a URL from which the model's config can be downloaded,
-            if necessary
-        weights_base_filename: the filename in ``fo.config.model_zoo_dir`` in
-            which to store the model's weights file
-        weights_base_gid: a google drive ID from which the model's weights can be
-            downloaded, if necessary
-        classes (None): an optional list of custom classes to use for zero-shot
-            prediction
+        nms_thresh (0.3) a non-maximum suppression threshold to use
     """
 
     def __init__(self, d):
         d = self.init(d)
         super().__init__(d)
 
+        self.config_url = self.parse_string(d, "config_url")
```

Comment:
Since you're subclassing from `TorchImageModel`, it was possible to reduce that amount of stuff explicitly declared here and instead rely on subclass implementation to parse the items.